docs/prod-fluo-setup.md

```diff
@@ -31,35 +31,49 @@ in `modules/distribution/target`:
 Install Fluo
 ------------
 
-When you have a distribution tarball built for your environment, follow these steps
-to install Fluo.
+After you obtain a Fluo distribution tarball, follow these steps to install Fluo.
 
-First, choose a directory with plenty of space and untar the distribution:
+1. Choose a directory with plenty of space and untar the distribution:
 
-    tar -xvzf fluo-1.0.0-beta-2-bin.tar.gz
+        tar -xvzf fluo-1.0.0-beta-2-bin.tar.gz
```

Comment:
Realized we are going to need to update docs with 1.0.0 tar name. I'll create an issue.